server/shared/descriptive.stats.js

```diff
@@ -55,3 +54,9 @@ export default function summaryStats(array) {
 		]
 	}
 }
+
+function computePercentile(values, percentile) {
+	const index = Math.abs((percentile / 100) * values.length - 1)
+	const value = Number.isInteger(index) ? (values[index] + values[index + 1]) / 2 : values[Math.ceil(index)]
+	return value
+}
```

Comment:
It would be better to import `computePercentile()` from `server/shared/compute.percentile.js` instead of duplicating it here. To prevent repeatedly sorting the array, you can add a `isSorted` parameter to the function. For example:
```
export default function computePercentile(values, percentile, isSorted) {
	if (!isSorted) values.sort((a, b) => a - b)
	const index = Math.abs((percentile / 100) * values.length - 1)
	const value = Number.isInteger(index) ? (values[index] + values[index + 1]) / 2 : values[Math.ceil(index)]
	return value
}
```

If you know that the input array is already sorted then you can set `isSorted=true`.